mailscanner/conf.php.example

```diff
@@ -145,7 +145,8 @@ define('SHOW_MORE_INFO_ON_REPORT_GRAPH', false);
 /* Mailwatch Interface Settings */
 define('MAILWATCH_MAIL_HOST', '127.0.0.1');
 define('MAILWATCH_MAIL_PORT', '25');
-define('MAILWATCH_FROM_ADDR', 'postmaster');
+// Change for your own email address
```

Comment:
No, it should be a valid email address, otherwise it might end up in spam.... 

There's no guarantee that that would yield a valid email address.